add a new client: WanChat (#3353)
matrix-org/matrix.org: content/ecosystem/clients/wanchat.md

+++
title = "WanChat"

[extra]
thumbnail = "wanchat.png"
maintainer = ""

> maturity = "Beta"

Reviewer: Given that this client is a fork of Element X, which is marked [`stable`](https://github.com/matrix-org/matrix.org/blob/93b60f42ad708253f5fa7f6c67f577ff4edf178e/content/ecosystem/clients/element-x.md?plain=1#L7), I'm not sure whether this client should inherit that label since the changes are rather minimal, or whether `beta` is more appropriate.

Author: Thanks for raising this point — it's a really good question about version semantics.
I've thought about it, and I agree with your first point: since this is a minimal fork that doesn't touch the core architecture, it's reasonable to inherit the stability of the upstream project (Element X).​ Marking it as "Stable" honestly reflects that users are getting essentially the same proven codebase, just with slight environmental adaptations.
I've gone ahead and updated maturityto "Stable"​ in the latest commit. This should set clearer expectations for users looking for a production-ready client.

repo = "https://github.com/lj92458/wanchat-android"
website = "https://wanchat.info"
matrix_room = ""
licence = "AGPL-3.0-only"
latest_release = "2026-04-12"
featured = false
[extra.features]
e2ee = true
spaces = true

> voip_1to1 = true

Reviewer: Given that Element X has [`voip_1to1`](https://github.com/matrix-org/matrix.org/blob/93b60f42ad708253f5fa7f6c67f577ff4edf178e/content/ecosystem/clients/element-x.md?plain=1#L20) marked as `false`, can you confirm that this fork does indeed support 1-to-1 voip?

Author: voip_1to1 = fase . Fixed

Reviewer: Quoted from a [previous comment](https://github.com/matrix-org/matrix.org/pull/3353#discussion_r3106372113):

> voip_1to1 = fase . Fixed

If that's the case, please update `wanchat.md`.

Author: done.

voip_jitsi = false
threads = false
sso = false
multi_account = false

> multi_language = true

Reviewer: What about support for `oauth`?

Author: yes,  fixed.

Reviewer: Quoted from a [previous comment](https://github.com/matrix-org/matrix.org/pull/3353#discussion_r3106369669):

> yes, fixed.

If that's the case, please update `wanchat.md`.

Author: done.

[extra.packages]
#google_play_store.app_id = ""
#f_droid.app_id = ""
+++

> A fork of Element X, adapted for China's environment. It can automatically delete records, clean room, install ntfy for notifications, and periodically wake up ntfy services.

Reviewer: I'm not too sure what is meant by "clean room" in this context.

Author: clean room，means delete all records of a room. thanks you!
Fixed.

Reviewer: Quoted from a [previous comment](https://github.com/matrix-org/matrix.org/pull/3353#discussion_r3106252991):

> clean room，means delete all records of a room. thanks you!
Fixed.

Thanks for the clarification. Given that, I'd welcome a reworking of the description.

Author: done.

+++
title = "WanChat"

[extra]
thumbnail = "wanchat.png"
maintainer = "WanChat"

> maturity = "Stable"

Reviewer: ```suggestion
maturity = "Beta"
```

Let's play it safe and label the client `Beta` for now; we can relabel it `Stable` after it's been used in the wild and proven reliable.

+++
title = "WanChat"

[extra]
thumbnail = "wanchat.png"
maintainer = "WanChat"

> maturity = "Stable"

Reviewer: I'm still not too sure which label is best here, so I'll have a further think and get back to you.

Author: welcome.

Reviewer: See this [comment](https://github.com/matrix-org/matrix.org/pull/3353#discussion_r3151416395).

repo = "https://github.com/lj92458/wanchat-android"
website = "https://wanchat.info"
matrix_room = ""
licence = "AGPL-3.0-only"
latest_release = "2026-04-12"
featured = false

> # featured_order = 1

Reviewer: ```suggestion
```

screenshots = []

> good_for = "Useful for China's environment, or users who require extremely high security"

Reviewer: ```suggestion
good_for = "Mobile-first users who want additional security (e.g., self-deleting messages). Tailored for China"
```

[extra.features]
e2ee = true
spaces = true
voip_1to1 = false
voip_jitsi = false
threads = false
sso = false
oauth = true
multi_account = false
multi_language = true
[extra.packages]
#google_play_store.app_id = ""

> #f_droid.app_id = ""

Reviewer: ```suggestion
```

+++

> A fork of Element X, adapted for China's environment. Auto delete records, delete all records of a room, install ntfy for notifications, and periodically wake up ntfy services.

Reviewer: ```suggestion
A fork of Element X, adapted for China's environment, that supports self‑destructing messages, removing all room records, and uses ntfy for push notifications.
```
